west.yml

```diff
@@ -21,12 +21,15 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: benpicco
+      url-base: https://github.com/benpicco
 
   #
   # Please add items below based on alphabetical order
   projects:
     - name: hal_atmel
-      revision: 04ff67a0826a51041e51034faf8fc4d3eeacd846
+      remote: benpicco
+      revision: 32ce0ddd1a3e0512de9822693e334d7090776ec0
```

Comment:
This should just be:

`revision: pull/1/head`

Reply:
Alright!
I didn't intent to commit this to master though 😉